### DISCUSSION TITLE
Help me understand why "Close Other Tabs" doest not work as I expect it to

### DISCUSSION THREAD

**Discussion by thesobercoder** [LazyVim/LazyVim#4091] [Q&A] [answered]
So I understand all tabs are essentially buffer in Neovim, but after opening multiple files in "tabs" when I press `<leader><tab>o` it always shows a message `already only one tab page`. Trying to understand what it means? Are these not tabs?

![CleanShot 2024-07-17 at 23 49 43](https://github.com/user-attachments/assets/871e7fb5-044e-4b04-87e0-090cd2b7f184)

**Comment by folke** [comment]
- A buffer is a file
- a window is literally a window that shows a buffer. (like a split has two windows)
- a tab is a layout of windows. I almost never use tabs myself. So always use a single tab.

The thing in red is a tabline, showing open buffers.

To get a better understanding of these concepts, it's best to read some blog posts about the topic.

**Comment by folke** [comment, marked as answer]
To close the other buffers, use `<leader>bo` instead

**Reply by thesobercoder** [reply]
Wow, that is so dumb of me. Thanks so much! I switched to Neovim recently, and it was only possible because of LazyVim and maintainers like you. The speed at which I can grep or fuzzy find files/symbols/texts is surreal sometimes. Kudos! Greatest community ever.

**Reply by newholder** [reply]
but why use this instead of `<leader><tab>o`. it's what in the documentation

**Reply by dpetka2001** [reply]
You should read `:h windows-intro` to understand better what Neovim considers buffers, windows and tabs. What you see at the top of OP's screenshot are actually buffers, not tabs. So using `<leader><tab>o` won't work for this.
